index.html

```diff
@@ -108,6 +108,19 @@
       </div>
     </form>
 
+    <!-- footer for repo link -->
```

Comment:
Hi @akhilkh2000, please remove this comment as it's not that complex part people will get it by looking at the <footer> tag

Reply:
Okay. I will do this